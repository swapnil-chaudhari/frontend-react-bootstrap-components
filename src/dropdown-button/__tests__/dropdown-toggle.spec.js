import React from 'react';
import renderShallow from 'render-shallow';
import DropdownToggle from '../dropdown-toggle';

describe('<DropdownToggle>', () => {
    describe('when it renders', () => {
        let component;

        beforeAll(() => {
            component = renderShallow(
                <DropdownToggle
                    withCaret={ false }
                    bsClass="btn"
                    title="Create"
                />
            ).output;
        });

        test('has proper markup', () => {
            expect(component).toMatchSnapshot();
        });
    });
});
