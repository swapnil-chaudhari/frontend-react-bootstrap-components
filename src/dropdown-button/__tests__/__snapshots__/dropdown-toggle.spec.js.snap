// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropdownToggle> when it renders has proper markup 1`] = `
<Button
  className="btn"
>
  Create
</Button>
`;
